From 9078f96867682386b4ac2f2d7fee6ea7c76b0517 Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 24 Oct 2014 17:32:27 -0400
Subject: [lib] rhashtable: remove second linux/log2.h inclusion

Message-id: <bfbad9753755d9b0074e0f0fd65f43fe3bd40e49.1414171887.git.jbenc@redhat.com>
Patchwork-id: 98438
O-Subject: [RHEL7.1 net PATCH 18/19] lib: rhashtable: remove second linux/log2.h inclusion
Bugzilla: 1140661
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Hannes Frederic Sowa <hannes@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1140661
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=8158749

commit b3f2512ecdb3561ffa44737f370fdb78e1febf6b
Author: Fabian Frederick <fabf@skynet.be>
Date:   Wed Sep 17 21:07:19 2014 +0200

    lib: rhashtable: remove second linux/log2.h inclusion

    linux/log2.h was included twice.

    Signed-off-by: Fabian Frederick <fabf@skynet.be>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/lib/rhashtable.c b/lib/rhashtable.c
index 7334786..081be3b 100644
--- a/lib/rhashtable.c
+++ b/lib/rhashtable.c
@@ -23,7 +23,6 @@
 #include <linux/hash.h>
 #include <linux/random.h>
 #include <linux/rhashtable.h>
-#include <linux/log2.h>
 
 #define HASH_DEFAULT_SIZE 64UL
 #define HASH_MIN_SIZE  4UL
-- 
1.7.1